import_manager.go

```diff
@@ -122,19 +121,7 @@ func recordImportWorker(id int, client *Client, field *Field, chans importWorker
 		errChan <- err
 	}()
 
-	rowKeyIDMap, err := lru.NewLRU(options.rowKeyCacheSize, nil)
-	if err != nil {
-		panic(errors.Wrap(err, "while creating rowKey to ID map"))
-	}
-	columnKeyIDMap, err := lru.NewLRU(options.columnKeyCacheSize, nil)
-	if err != nil {
-		panic(errors.Wrap(err, "while creating columnKey to ID map"))
-	}
-	state := &importState{
-		rowKeyIDMap:    rowKeyIDMap,
-		columnKeyIDMap: columnKeyIDMap,
-	}
-
+	state := &importState{}
```

Comment:
I'm not sure I fully understand this. It looks like the impact of this is that we don't create the caches until the first time a given worker uses them, but other than that, it should behave the same (reusing them for the lifetime of that worker, etc.)?

Reply:
This allows us to postpone cache creation is postponed until it is required, like rowKey cache is only created for `RowKeyColumnID` or `RowKeyColunKey` type data; it's similar for columnKey cache. Related code is in `importColumnsRoaring` in `client.go`.